sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/java/com/azure/ai/inkrecognizer/ApplicationKind.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Is this 100% a source code move from one repo to the central Java SDK repo, or were there any code changes made here?

Reply:
I did make some changes here to get the validation checks to succeed and also to address comments.